java/component/keyple-plugin/pcsc/src/main/java/org/eclipse/keyple/plugin/pcsc/PcscReader.java

```diff
@@ -11,10 +11,144 @@
  ************************************************************************************** */
 package org.eclipse.keyple.plugin.pcsc;
 
+import javax.smartcardio.Card;
+import org.eclipse.keyple.core.seproxy.SeReader;
 import org.eclipse.keyple.core.seproxy.event.ObservableReader;
+import org.eclipse.keyple.core.seproxy.protocol.TransmissionMode;
 
 /**
- * The PcscReader interface remains internal since it does not define any complement to {@link
- * ObservableReader}.
+ * Interface extending {@link ObservableReader}) and allowing to set the operating parameters of a
+ * reader based on a PC/SC hardware interface (USB CCID).
+ *
+ * @since 1.0
  */
-interface PcscReader extends ObservableReader {}
+public interface PcscReader extends ObservableReader {
+
+  /**
+   * Defines the expected behavior regarding the access to the SE in a
+   * multithreaded/multi-application context.
+   *
+   * <p>Corresponds to the beginExclusive() and endExclusive() methods of smartcard.io and, at a
+   * lower level, to the connection mode defined by PC/SC and used in the SCardConnect function.
+   *
+   * @since 1.0
+   */
+  enum SharingMode {
+    /** Allows simultaneous access to the SE */
+    SHARED,
+    /** Requests exclusive access to the SE */
+    EXCLUSIVE
+  }
+
+  /**
+   * Available transmission protocols as defined in the PC/SC standard.
+   *
+   * @since 1.0
+   */
+  enum IsoProtocol {
+    /** to connect using any available protocol */
+    ANY("*"),
+    /** to connect using T=0 protocol */
```

Comment:
Tu pourrais donc par exemple écrire ici /** To connect using {@value} protocol */
De cette façon ça évite les doublons, et donc le risque d'erreur.
Mais je suis vraiment pas sûr que ça marche.

Reply:
Ok